model/sleep_state.go

```diff
@@ -28,3 +33,24 @@ type SleepState struct {
 type SleepStateTimeout struct {
 	StateExecTimeout StateExecTimeout `json:"stateExecTimeout,omitempty"`
```

Comment:
If the StateExecTimeout is optional, we should change it's type to *StateExecTimeout？

Reply:
the
```
Timeouts *SleepStateTimeout
```

already is a pointer, is there a need to use pointers again here?